scripts/test_e2ee_integration.py

```
@@ -4,58 +4,55 @@
+    """Raised when Matrix connection fails."""
 
-E2EEDebugUtilities = (
-    _ImportedE2EEDebugUtilities if IMPORTS_AVAILABLE else _FallbackE2EEDebugUtilities
-)
+    pass
 
 
 class E2EEIntegrationTester:
     """Integration tester for E2EE functionality"""
 
     def __init__(self):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)